src/Topics/StandardModernPrecision/OneDiamondResponses.hs

```diff
@@ -97,14 +97,14 @@ reverseFlannery = let
         action = do
             b1D
             oppsPass
-        explanation fmt =
-            "With 5 spades, 4 or 5 hearts, and " ++
-            (if isInvite then "" else "less than ") ++
-            "invitational strength, bid a Reverse Flannery " ++
-            displayLastCall fmt bid ++ ". Partner can then place the final\
-          \ contract, bid " ++ output fmt (T.Bid 3 T.Clubs) ++ " (pass or\
-          \ correct) with both minors, or bid " ++
-            output fmt (T.Bid 2 T.Notrump) ++ " to ask for\
+        explanation =
+            "With 5 spades, 4 or 5 hearts, and " .+
+            (if isInvite then "" else "less than ") .+
+            "invitational strength, bid a Reverse Flannery " .+
+            displayLastCall bid .+ ". Partner can then place the final\
+          \ contract, bid " .+ T.Bid 3 T.Clubs .+ " (pass or\
+          \ correct) with both minors, or bid " .+
+            T.Bid 2 T.Notrump .+ " to ask for\
```

Comment:
I should find a way to format multi-line strings that I really like, and line-wrap everything properly. That should happen when I'm done fiddling around, which isn't yet. but in the meantime I should at least see if there's some standard idiom in the wider community for multi-line strings.